addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -69,6 +87,7 @@
 				<font>-</font>
 				<texturefocus>OSDRewindFO.png</texturefocus>
 				<texturenofocus>OSDRewindNF.png</texturenofocus>
+				<animation effect="fade" start="100" end="50" time="100" condition="!Player.SeekEnabled">Conditional</animation>
```

Comment:
Can you also add <enable>Player.SeekEnabled</enable> here and on other buttons with effect, without it the button can be selected.
I create currently a pull request (was before in RDS request) to add pvr to music osd.